Sources/OpenAI/Client.swift

```diff
@@ -827,6 +830,40 @@ public final class Client {
                 }
             }
     }
+    
+    // MARK: Content Filter
+    /**
+     The filter aims to detect generated text that could be sensitive or unsafe coming from the API. It's currently in beta mode and has three ways of classifying text- as safe `0`, sensitive `1`, unsafe `2`, or error `-1`. The filter will make mistakes and we have currently built it to err on the side of caution, thus, resulting in higher false positives.
```

Comment:
Nitpick: [Semantic Line Breaks](https://sembr.org/)

```suggestion
     The filter aims to detect generated text that could be sensitive or unsafe coming from the API. 
     It's currently in beta mode and has three ways of classifying text - 
     as safe `0`, sensitive `1`, unsafe `2`, or error `-1`. 
     The filter will make mistakes and we have currently built it to err on the side of caution, 
     thus, resulting in higher false positives.
```